src/test/java/io/confluent/examples/streams/FanoutLambdaIntegrationTest.java

```diff
@@ -72,6 +72,7 @@ public void shouldFanoutTheInput() {
     streamsConfiguration.put(StreamsConfig.APPLICATION_ID_CONFIG, "fanout-lambda-integration-test");
     streamsConfiguration.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "dummy config");
     streamsConfiguration.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
+    streamsConfiguration.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.ByteArray().getClass().getName());
```

Comment:
Weird, this test really did use the ByteArraySerde. Tbh maybe we should just convert it to a more human readable type, like a String or Integer or something? Not sure how much work that would be, but if it's just ~10 LOC then let's just clean up the test

If it would be a big overhaul then I'm fine with just explicitly setting it to the ByteArraySerde, it just seems bizarre to actually be using this, especially in an example

Reply:
I just switched it to be string and it didn't complain 🤔  I think because it's using the topology test driver we need the config but if you look further down in the test it looks like we're passing in `new IntegrationTestUtils.NothingSerde<>()`

Reply:
I guess the StringSerde is also not too picky, since it's just an arbitrary length of bytes same as the byte array. It would be gibberish though, the test is clearing _intending_ to use the ByteArraySerde since the types are things like `KStream<byte[], String>`

If we really don't care at all about the key, and maybe we don't, then I would just keep it as a ByteArraySerde. Otherwise it's confusing since users see that it's a StringSerde in the configs, but the KStream type shows it as a `byte[]`

Reply:
It works with a string serde too.. It seems like because it uses the topology test driver we set the serdes earlier? So it requires the config to be set when we initialize the topology test driver